gradoop-data-integration/src/main/java/org/gradoop/dataintegration/importer/csv/MinimalCSVImporter.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.dataintegration.importer.csv;
+
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.flink.api.java.DataSet;
+import org.apache.flink.api.java.utils.DataSetUtils;
+import org.gradoop.common.model.impl.properties.Properties;
+import org.gradoop.flink.io.impl.graph.tuples.ImportVertex;
+import org.gradoop.flink.util.GradoopFlinkConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Read a csv file and import each row as a vertex in EPGM representation.
+ */
+public class MinimalCSVImporter {
+
+  /**
+   * Log if an error by reading of the file occurs.
+   */
+  private static final Logger LOG = LoggerFactory.getLogger(MinimalCSVImporter.class);
+
+  /**
+   * Token delimiter
+   */
+  private String tokenSeparator;
+
+  /**
+   * Path to the csv file
+   */
+  private String path;
+
+  /**
+   * The charset used in the csv file.
+   */
+  private String charset;
+
+  /**
+   * THe property names for all columns in the file.
+   */
+  private List<String> columnNames;
+
+  /**
+   * Gradoop Flink configuration
+   */
+  private GradoopFlinkConfig config;
+
+  /**
+   * Create a new MinimalCSVImporter. The user set a list of the property names.
```

Comment:
Put a blank line between description and params. Same for other constructors and functions.